packages/react-storage/src/components/StorageManager/StorageManager.tsx

```diff
@@ -84,6 +86,14 @@ function StorageManager({
     setUploadResumed,
   } = useStorageManager(defaultFiles);
 
+  React.useImperativeHandle(filesRef, () => {
+    return {
+      clearFiles: () => {
+        clearFiles();
+      },
+    };
+  });
```

Comment:
Nit: can be simplified to:
```
React.useImperativeHandle(ref, () => ({ clearFiles }));
```

Reply:
Ack. Will fix in next commit with additional changes. Thanks!!